VMware.CloudFoundation.Reporting.psm1

```diff
@@ -824,7 +824,159 @@ Function Request-SoSHealthJson {
         JSON file to the local file system.
 
         .EXAMPLE
```

Comment:
Can you add a example for a workload domain, too. e.g., `-workloadDomain sfo-w01`

Reply:
☝️ Reminder to add the workload domain example, please.